### DISCUSSION TITLE
war files from being decompiled

### DISCUSSION THREAD

**Discussion by Jie-YangZeng** [saphirepankaj/jexec-build#3] [Q&A] [answered]
Dear GitHub contributor,

Hello, I hope this message finds you well. I noticed that you have created a project aimed at preventing jar file decompilation, which I find quite impressive. I was wondering if you also have any recommendations or methods to prevent war files from being decompiled? I would greatly appreciate your insights on this matter. Thank you for your time and expertise.

Thank you for your time and expertise.

Best regards,
Jie-YangZeng

**Comment by saphirepankaj** [comment, marked as answer]
Hi Jie-YangZeng, we are working on preventing war files from de-compilation as well. It's not quite straight forward, so, we'll update you once done.
